test/python_output9.py

```
@@ -0,0 +1,2 @@
+data = open('new_file')
+requests.put('http://awesomeurl.com/upload', data=data)
```

Ruff (0.11.9)

Undefined name `requests`

(F821)